tensorflow_addons/custom_ops/image/python/dense_image_warp_test.py

```diff
@@ -0,0 +1,270 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for dense_image_warp."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import math
+import numpy as np
+
+
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import errors
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import test_util as tf_test_util
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import gradients
+from tensorflow.python.ops import math_ops
+from tensorflow.python.ops import random_ops
+from tensorflow.python.ops import variables
+from tensorflow.python.platform import test
+from tensorflow.python.training import adam
+from tensorflow_addons.custom_ops.image.python import dense_image_warp as dense_image_warp_ops
+
+
+class DenseImageWarpTest(test.TestCase):
+
+    def setUp(self):
+        np.random.seed(0)
+
+    @tf_test_util.run_all_in_graph_and_eager_modes
+    def test_interpolate_small_grid_ij(self):
+        with self.cached_session():
```

Comment:
Can you remove all session in those tests?

Reply:
Except for `test_gradients_exist`, which runs on v1 only, I've removed them.

Reply:
Great! That's fine for me.